hpobench/benchmarks/nas/nasbench_1shot1.py

```diff
@@ -0,0 +1,309 @@
+"""
+Interface to Benchmarks of Nasbench 1shot 1
+https://github.com/automl/nasbench-1shot1/tree/master/nasbench_analysis/
+
+
+How to use this benchmark:
+--------------------------
+
+We recommend using the containerized version of this benchmark.
+If you want to use this benchmark locally (without running it via the corresponding container),
+you need to perform the following steps.
```

Comment:
Please add a note, that this is a very memory intensive benchmark

Reply:
Since it does rely on the nasbench101, and not on the 201 as I first thought (sorry :-) ), it is not that memory-consuming any more.

Reply:
Okay, sounds even better